src/D2L.CodeStyle.Analyzers/Diagnostics.cs

```diff
@@ -13,7 +13,7 @@ public static class Diagnostics {
 			category: "Correctness",

```

Comment:
Fixed all the straight forward ones, but this is the sort of thing it warns about:

![image](https://user-images.githubusercontent.com/12101120/151646825-d7584dc9-751c-4a48-a60f-9c13959acb61.png)